src/odoc/odoc_file.mli

```diff
@@ -30,6 +30,11 @@ type content =
 
 type t = { content : content; warnings : Error.t list }
 
+type 'a index = {
+  sidebar : Odoc_index.Sidebar.t;
+  index : 'a Paths.Identifier.Hashtbl.Any.t;
+}
+
```

Comment:
This is temporary to deal with the deletion of `Lang.Index`

Reply:
It would be more logic to have this in `src/index/odoc_index.ml` (which should then not be removed, contrary to my previous comment).

Indeed, that's where it is going to be replaced by the new index type.